src/cephadm/build.py

```diff
@@ -269,6 +269,9 @@ def _build(dest, src, config):
         mdir.mkdir(parents=True, exist_ok=True)
         (mdir / "__init__.py").touch(exist_ok=True)
         versioning_vars = config.cli_args.version_vars
+        shutil.copytree(
+            "../python-common/ceph", appdir / "ceph"
+        )
```

Comment:
IMO this should be a separate commit that is clearly described in the commit message: as incorporating the ceph package from python-common into the cephadm binary. This is a significant enough change (in scope, not lines of code) that it deserves to be called out in the history.

Reply:
put this change in its own commit

Reply:
@adk3798 as this location is for cephadm only shouldn't we create a cephadm subdirectory to make it clear instead of including the whole ceph dierctory?

Reply:
I think there's a chance in the future we'll end up wanting something from python-common/ceph/deployment and including the extra stuff in the zip doesn't have any downsides I'm aware of (I guess technically it takes up more space, but we're talking about kilobytes). That was my reasoning for just including the whole thing.

Reply:
@adk3798 sounds good, we just have to keep an eye on this directory in case somebody else put some big new module inside :+1:

Reply:
Including python-common/ceph is a foundational change that allows cephadm binary and the other parts of ceph's python codebase to share code. That hypothetical module that we would want to exclude would have to be pretty huge for us to notice a problem IMO.

Reply:
@phlogistonjohn yeah, probably we should setup something so any change on the ` python-common/ceph` would include the `orchesrtators` team as reviewers ... though I'm not sure if that would help or not .. just something to consider.

Reply:
Fair enough. I have even wondered if a more general `python-sig` should exist, but for now that would be good enough. That doesn't need to be part of this PR however. :-)

Reply:
https://github.com/ceph/ceph/pull/60546